<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1078</width>
    <height>771</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Логика</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Ответ тренера</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="process_btn">
            <property name="text">
             <string>Обработка</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="clear_answer">
            <property name="text">
             <string>Очистить</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="teacher_answer_te">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="plainText">
           <string>Вставьте сюда ответ тренера
</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="copy_my_answer_btn">
          <property name="text">
           <string>Копировать мой ответ</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="my_answer_te">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>10</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Неверное решение</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="run_incorrect_btn">
            <property name="text">
             <string>Run</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTabWidget" name="incorrect_answer_tw">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string>Текст</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QPlainTextEdit" name="incorrect_answer_te">
              <property name="font">
               <font>
                <family>Consolas</family>
                <pointsize>10</pointsize>
               </font>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_2">
           <attribute name="title">
            <string>Строки</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QTableView" name="incorrect_answer_tv">
              <property name="font">
               <font>
                <family>Consolas</family>
                <pointsize>10</pointsize>
               </font>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="incorrect_result">
          <property name="text">
           <string>Результат</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Объяснение</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="explanation_te">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>10</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Верное решение</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pep8_btn">
          <property name="text">
           <string>PEP8</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="run_correct_btn">
          <property name="text">
           <string>Run</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTabWidget" name="correct_answer_tw">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab_5">
         <attribute name="title">
          <string>Текст</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QPlainTextEdit" name="correct_answer_te">
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>10</pointsize>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_6">
         <attribute name="title">
          <string>Строки</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QTableView" name="correct_answer_tv">
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>10</pointsize>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="correct_result">
        <property name="text">
         <string>Результат</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
